torch/distributed/_fsdp/wrap.py

```diff
@@ -264,6 +269,11 @@ def recursive_wrap(module: nn.Module, auto_wrap_policy: Optional[Callable], **kw
                 module to recursively wrap
             auto_wrap_policy (Callable, Optional):
                 optionally, override the :func:`auto_wrap_policy` from the context.
+            only_wrap_children (bool, Optional):
```

Comment:
nit: curious, under the current plan, what belongs to `auto_wrap_policy`, and what should be separate args like `only_wrap_children`?

Reply:
this arg is for private `ConfigAutoWrap._recursive_wrap` API which should not be used directly by users. So under the current plan, users only have to pass in the policy, something like

```
module = FSDP(module, auto_wrap_policy=partial(default_policy, min_num_params=1e7))
```